sequencer/src/api/options.rs

```diff
@@ -31,6 +38,7 @@ pub struct Options {
     pub submit: Option<Submit>,
     pub status: Option<Status>,
     pub state: Option<State>,
+    pub merklized_state: Option<MerklizedState>,
```

Comment:
IMO: rename this to `state` and rename  `state` to `catchup` which is a better name for it anyways